lib/bundler/remote_specification.rb

```diff
@@ -75,11 +75,7 @@ def _remote_specification
     end
 
     def method_missing(method, *args, &blk)
-      if Gem::Specification.new.respond_to?(method)
-        _remote_specification.send(method, *args, &blk)
-      else
-        super
-      end
+      _remote_specification.send(method, *args, &blk)
```

Comment:
I'm pretty sure that simply calling `_remote_specification` will instantiate it, and this was trying to avoid instantiating it if it didn't respond to the method that was being called? but maybe we removed whatever made that relevant.